DOCUMENTATION.md

```diff
@@ -10,83 +29,86 @@ If you want to read more into Flatseal's permissions, you can look at the [Permi
-Other files | Input | Allow or prohibit read-write access to the application to the directory you desire. <br /> <br /> For example, you would put `~/games` if you want read-write access to `~/games`. If you want read-only (`ro`) access to `~/games`, then it will be `~/games:ro`. | `--filesystem=[PATH]`, `--filesystem=[PATH]:ro` and `--nofilesystem=[PATH]`
+All filesystem files | Toggle | Allow read-write access to the application to the whole filesystem. Everything that isn't writeable by the user will be read-only | `--filesystem=host` and `--nofilesystem=host`
+All system libraries, executables and static data | Toggle | Allow read-write access to the application to system libraries located in `/usr`. Since this directory requires root access to write, the permission will be read-only. | `--filesystem=host-os` and `--nofilesystem=host-os`
+All system configurations | Toggle | Allow read-write access to the application to system configurations located in `/etc`. Since this directory requires root access to write, the permission will be read-only. | `--filesystem=host-etc` and `--nofilesystem=host-etc`
+All user files | Toggle | Allow read-write access to the application to the user directory (`$HOME` or `~/`). | `--filesystem=home` and `--nofilesystem=home`
+Other files | Input | Allow read-write access to the application to the directory you desire. <br /> <br /> For example, you would put `~/games` if you want read-write access to `~/games`. If you want read-only access to `~/games`, then you would put `~/games:ro`. | `--filesystem=[PATH]`, `--filesystem=[PATH]:ro` and `--nofilesystem=[PATH]`
 
 #### Persistent
 
 List of the homedir-relative paths created in the sandbox.
 
 Name | Type | Description | `flatpak-override` equivalent
 --- | --- | --- | ---
-Files | Input | Allow only to the application to have access to the targeted directory while restricting other applications from accessing it. <br /> <br /> Starting from `$HOME`, the targeted directory will be remapped to the application's directory (`~/.var/app/$FLATPAK_APP_ID/[PATH]`) if it has no write access to the targeted directory. <br /> <br /> For example, persisting `.mozilla` will map `~/.mozilla` to `~/.var/app/org.mozilla.Firefox/.mozilla` if the Firefox Flatpak has no write access to `~/.mozilla`. | `--persist=[PATH]`
+Files | Input | Allow only to the application to have access to the targeted directory while restricting other applications from accessing it. <br /> <br /> Starting from the user directory (`$HOME` or `~/`), the targeted directory will be remapped to the application's directory (`~/.var/app/$FLATPAK_APP_ID/[PATH]`) if it has no write access to the targeted directory. <br /> <br /> For example, persisting `.mozilla` will map `~/.mozilla` to `~/.var/app/org.mozilla.Firefox/.mozilla`. <br /> <br /> This is also a technique used to declutter the user directory, as it prevents the application from writing to `~/`. | `--persist=[PATH]`
```

Comment:
This require more passes, I think is still not sufficiently clear.

Reply:
I honestly think that it's clear. Do you have any suggestion?

Reply:
I think there should be a more direct way of explaining what it does and at the same time doing it with less technical details.

Reply:
> For example, persisting `.mozilla` will map `~/.mozilla` to `~/.var/app/org.mozilla.Firefox/.mozilla`

I honestly think that this example is a good example and explains it really well.

Reply:
The example, yes. But the description, really needs more work.

Reply:
> Allow the application to only to access to the targeted directory while restricting other applications from accessing it.

I honestly don't like this description. It's unclear and isn't exactly true.

Reply:
> Starting from the user directory (`$HOME` or `~/`), the targeted directory will be remapped to the application's directory (`~/.var/app/$FLATPAK_APP_ID/[PATH]`) if it has no write access to the targeted directory. <br /> <br /> For example, persisting `.mozilla` will map `~/.mozilla` to `~/.var/app/org.mozilla.Firefox/.mozilla`

As for those, I don't know how to make them any simpler. The details are there: anything the user writes in the box will start from `~/`, the targeted directory moves to `~/.var/app/$FLATPAK_APP_ID/[PATH]` if it has no access to the targeted directory.